### PR TITLE
Rewrote a gettingStarted pages

### LINKED ISSUE #393: Change the Installation page to improve onboarding experience

### PR DESCRIPTION
**Pull request by zaleslaw**
Fixes #393

### CODE REVIEW

**Comment by zaleslaw** [comment]
I am trying to keep all the useful information, updated the plugin info in the gradle, most of the phrases and style was copied from the Kotlin documentation site.

File: docs/StardustDocs/d.tree [hunk 1]
```diff
@@ -25,7 +30,6 @@
             <toc-element topic="schemasImportOpenApiJupyter.md"/>
         </toc-element>
     </toc-element>
-    <toc-element topic="installation.md"/>
```
Comment (koperagen) [inline docs/StardustDocs/d.tree:28]: I think it's likely that we refer to this somewhere (notebook examples, videos, blogposts), did you check with global search in ide? Anyway, we should configure and test redirect https://github.com/Kotlin/dataframe/blob/master/docs/StardustDocs/redirection-rules.xml

**Review by koperagen — CHANGES_REQUESTED** [review]

File: docs/StardustDocs/topics/gettingStarted.md [hunk 2]
```diff
@@ -2,157 +2,85 @@
...
+
+1. **Create your first application for data exploration:**
+   * To start from scratch,
+   create a [basic JVM application with the IntelliJ IDEA project wizard](gettingStartedGradle.md).
+   * To set up the custom Gradle configuration or Linter configuration, follow the instruction on the [page](gettingStartedGradleAdvanced.md).
+   * If you prefer more robust examples, try to use Kotlin DataFrame
+   together with KotlinDL like in the [Titanic example](https://github.com/Kotlin/dataframe/tree/master/examples/idea-examples/titanic).
+2. **Use Kotlin DataFrame and third-party Kotlin Data Science libraries in your application.**
+   * Find the curated list of the recommended science libraries on the JVM [here](https://kotlinlang.org/docs/data-science-overview.html#java-libraries).
+3. **Learn more about Kotlin DataFrame usage:**
```
Comment (koperagen) [inline docs/StardustDocs/topics/gettingStarted.md:28]: i was quite surprised that this page leads to a big blogpost with examples, video and bunch of links. Maybe it's worth it to pull out a video from there and embed it to the page, or provide short overview of the content?

File: docs/StardustDocs/topics/gettingStarted.md [hunk 3]
```diff
@@ -2,157 +2,85 @@
...
+With Datalore, you can use Kotlin in the browser straight out of the box, no installation required. 
+
+You can also collaborate on Kotlin notebooks in real time, 
+get smart coding assistance when writing code, and share results as interactive or static reports.
+
+1. **[Create your first Notebook in Datalore](https://www.jetbrains.com/datalore/features/notebooks/)**
+    * To start from scratch, create a [basic Kotlin Notebook](gettingStartedDatalore.md).
+    * If you prefer more robust examples, download from the GitHub, upload them to Datalore
+      and run [these examples](https://github.com/Kotlin/dataframe/tree/master/examples/notebooks) here.
+2. **[Learn more about Kotlin Kernel for Jupyter Notebook.](https://github.com/Kotlin/kotlin-jupyter)**
```
Comment (Jolanrensen) [inline docs/StardustDocs/topics/gettingStarted.md:75]: the Kotlin kernel for Jupyter notebooks

File: docs/StardustDocs/topics/gettingStarted.md [hunk 4]
```diff
@@ -2,157 +2,85 @@
...
+3. **Join the Kotlin Data Science community:**
+    * ![Slack](https://kotlinlang.org/docs/images/slack.svg)Slack:
+      get an [invite](https://surveys.jetbrains.com/s3/kotlin-slack-sign-up?_ga=2.148899126.1808346675.1686912551-1605045648.1686912543&_gl=1*64xzk9*_ga*MTYwNTA0NTY0OC4xNjg2OTEyNTQz*_ga_9J976DJZ68*MTY4NjkxMjU0Mi4xLjEuMTY4NjkxNTE3OC41OC4wLjA.)
+      and join the #datascience, #science, #mathematics.
```
Comment (Jolanrensen) [inline docs/StardustDocs/topics/gettingStarted.md:79]: ... channels

File: docs/StardustDocs/topics/gettingStartedDatalore.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
+[//]: # (title: Get started with Kotlin DataFrame on Datalore)
+
+## Datalore
+
+To start with the Kotlin DataFrame library in Datalore, create a Kotlin notebook first:
+
+![Installation to Datalore](datalore-1.png)
+
+As the Notebook you've created is a Jupyter notebook, you can follow the instructions
+in the [previous section](gettingStartedJupyterNotebook.md) to turn the Kotlin DataFrame library on.
```
Comment (Jolanrensen) [inline docs/StardustDocs/topics/gettingStartedDatalore.md:10]: activate the Kotlin DataFrame library?

File: docs/StardustDocs/topics/gettingStartedJupyterNotebook.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
+[//]: # (title: Get started with Kotlin DataFrame on Jupyter Notebook)
+
+## Jupyter Notebook
+
+You can use the Kotlin DataFrame library in Jupyter Notebook and in Jupyter Lab.
```
Comment (Jolanrensen) [inline docs/StardustDocs/topics/gettingStartedJupyterNotebook.md:5]: notebooks

File: docs/StardustDocs/d.tree (hunk 1 above)
Comment (zaleslaw) [reply docs/StardustDocs/d.tree:28]: First question: what is the global search in IDE?
The second question: where to read about redirection?

IMHO, we don't need to be so patient to that before we release stable and full covered documentation

File: docs/StardustDocs/topics/gettingStartedJupyterNotebook.md (hunk 6 above)
Comment (zaleslaw) [reply docs/StardustDocs/topics/gettingStartedJupyterNotebook.md:5]: Both of them are product names, and we mean these products, not just notebooks. If only notebook is mentioned, but Jupyter notebook looks weird like jupyter Notebook

File: docs/StardustDocs/d.tree (hunk 1 above)
Comment (koperagen) [reply docs/StardustDocs/d.tree:28]: https://plugins.jetbrains.com/plugin/20158-writerside/docs/place-redirects.html
I don't mind other topics such as operations etc, but installation page specifically should be redirected. It won't take too much effort

File: docs/StardustDocs/topics/gettingStartedJupyterNotebook.md (hunk 6 above)
Comment (Jolanrensen) [reply docs/StardustDocs/topics/gettingStartedJupyterNotebook.md:5]: If you're referring to the product, then indeed "Jupyter Notebook". But how often do you say it out loud like that in a sentence? Sounds like an Apple presentation if you say "You can run your Jupyter Notebook (TM) however you like!", compared to "You can run any of your Jupyter notebooks however you like".
I think more often, you are referring to notebooks the concept than the actual name.

**Pull request merged**